test-files/cmd/glacial/NonDestructiveConflationTest.sh

```diff
@@ -67,5 +75,12 @@ hoot changeset-derive --stats \
 # | Create |  859 |        0 | 309 |
 # | Delete |    1 |        1 |  17 |
 # | Modify |    0 |        0 |   9 |
-
+#
+# @ - WaySplitter::createSplits change
+#
+# |        | Node | Relation | Way |
+# | Create |  859 |        0 | 282 |
+# | Delete |    1 |        1 |  17 |
+# | Modify |    0 |        0 |   9 |
+#
```

Comment:
This ended up being a great change to the `NonDestructiveConflationTest` in that the number of new ways was reduced by 27.  We are now less that the `REF1 way count + REF2 way count` sum (300 ways).

Reply:
awesome